src/Common/src/Common/Logging/BootstrapLoggerFactory.cs

```diff
@@ -4,7 +4,12 @@
 
 namespace Steeltoe.Common.Logging;
 
-internal static class BootstrapLoggerFactory
+/// <summary>
+/// Provides access to logging infrastructure before service container is created. Any loggers created are upgraded with config settings after
+/// config subsystem and later DI subsystem are available.
+/// This class should only be used by components that need logging logging infrastructure before service container is available.
+/// </summary>
```

Comment:
"service container" seems odd.  If it's a singleton, would Service Container be better?  Or perhaps "the service container."